sample-apps/generic_deepgrow/README.md

```diff
@@ -2,9 +2,73 @@
 
 ### Model Overview
 
-Interactive MONAI Label App using DeepGrow (https://arxiv.org/abs/1903.08205).
+This MONAI Label App is based on DeepGrow which allows for an interactive segmentation, where 
+the user can guide the segmentation using positive and negative clicks (https://arxiv.org/abs/1903.08205).
 It uses pre-trained Deepgrow Models for NVIDIA Clara.
 
+To those extending this App, please clone the folder and adjust the following hyper-parameters:
```

Comment:
Should we avoid using the word cloning for this step? There is a CLI command for this 

> monailabel apps --name generic_deepgrow --download --output myapp

Reply:
let me fix this.